src/platform_impl/macos/menu_item.rs

```diff
@@ -54,7 +56,7 @@ impl TextMenuItem {
 
     pub fn set_label(&mut self, label: impl AsRef<str>) {
         unsafe {
-            let title = NSString::alloc(nil).init_str(label.as_ref());
+            let title = NSString::alloc(nil).init_str(&remove_mnemonic(label));
```

Comment:
Since we are removing the mnemonic when setting the `label`, we should add it back when returning the label in `TextMenuItem::label`

Reply:
Seems It should be store somewhere in the `TextMenuItem` struct? Since ampersands were removed from the `NSMenuItem`, we can't retrieve from it.

Something like this
```rust
pub struct TextMenuItem {
    pub(crate) id: u64,
    pub(crate) ns_menu_item: id,
    label: String,
}
```

Reply:
yeah that sounds fine, although you want to wrap the label in `Rc` because `TextMenuItem` derives `Clone` and we want the clone to also point to the same label.